### PR TITLE
Update prepare_trials_columns() return type

### PR DESCRIPTION
**Pull request by raphant**
## Summary
HyperoptTools.prepare_trials_columns() was set to wrong return type

### Quick Changes
Imported pandas in `hyperopt_tools.py` and set `HyperoptTools.prepare_trials_columns()` return type to `pd.DataFrame`

### CODE REVIEW

File: freqtrade/optimize/hyperopt_tools.py [hunk 1]
```diff
@@ -298,7 +299,7 @@ def _format_explanation_string(results, total_epochs) -> str:
                 f"Objective: {results['loss']:.5f}")
 
     @staticmethod
-    def prepare_trials_columns(trials, legacy_mode: bool, has_drawdown: bool) -> str:
+    def prepare_trials_columns(trials, legacy_mode: bool, has_drawdown: bool) -> pd.DataFrame:
```
Comment (xmatthias) [inline freqtrade/optimize/hyperopt_tools.py]: I think you should also type "trials" then ... to make it complete (that's why it wasn't captured as part of mypy in the first place).

Comment (raphant) [reply freqtrade/optimize/hyperopt_tools.py]: Hmm, not sure I understand. Where would I type "trials"?

Comment (xmatthias) [reply freqtrade/optimize/hyperopt_tools.py]: What i mean is type the parameter (as follows): 

`def prepare_trials_columns(trials: pd.DataFrame, ...`  

It's a completeness topic (but so is having the right return type) :D

Comment (raphant) [reply freqtrade/optimize/hyperopt_tools.py]: Ah yes! Good point :)

**Review by xmatthias — APPROVED** [review]
LGTM :+1:

**Pull request merged**